schematic/store/synapse.py

```diff
@@ -250,25 +251,28 @@ def getDatasetManifest(self, datasetId: str, downloadFile: bool = False) -> List
         Returns:
             A tuple of manifest file ID and manifest name -- (fileId, fileName); returns empty list if no manifest is found.
             (or)
-            synapseclient.entity.File: A new Synapse Entity object of the appropriate type.
+            synapseclient.entity.File: A new Synapse Entity object of the appropriate type, if downloadFile is set to True
         """
 
         # get a list of files containing the manifest for this dataset (if any)
-        manifest = self.getFilesInStorageDataset(datasetId, fileNames = [os.path.basename(self.manifest)])
-
-        if not manifest:
+        all_files = self.storageFileviewTable
+        manifest = all_files[(all_files["name"] == os.path.basename(self.manifest)) & (all_files["parentId"] == datasetId)] 
```

Comment:
I'm going to test this function locally, but I wanted to issue this review in the interim.